src/components/data-management/ProjectDetails.jsx

```diff
@@ -589,10 +593,41 @@ const ProjectDetails = ({ width, height }) => {
           Processed Seurat object (.rds)
         </Tooltip>
       </Menu.Item>
-      <Menu.Item disabled>
-        <Tooltip title='Coming soon!' placement='left'>
-          Data Processing settings
-        </Tooltip>
+      <Menu.Item
+        onClick={async () => {
+          const experiment = activeProject.experiments[0];
+          const experimentName = experiments[experiment]?.name
+
+          // load processing configuration
+          await dispatch(loadProcessingSettings(experiment));
```

Comment:
Should you await this? Also I think its better to load the settings in a useEffect, this way you don't dispatch the action on every onClick which is pointless.

Reply:
You are correct, I've changed to `useEffect`